### PR TITLE
[RFR] Authentication for api (admin/api)

### PR DESCRIPTION
**Pull request by djhi**
- [x] Fixed babelrc
- [x] Added `api` and `admin` entry points on server
- [x] Added jwt token verification middleware
- [x] Added authentication routes
- [x] Added commands to create users (roles are not handled for now)
- [x] Fixed db config

### CODE REVIEW

File: api/src/users/authenticateRoutes.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import config from 'config';
+import koa from 'koa';
+import koaRoute from 'koa-route';
+import jwt from 'jsonwebtoken';
+import userRepositoryFactory from './userModel';
+
+const app = koa();
+let userRepository;
+
+app.use(function* init(next) {
```
Comment (Kmaschta) [inline api/src/users/authenticateRoutes.js]: What is the point of this middleware ?

**Comment by Kmaschta** [comment]
:+1:

File: api/src/users/authenticateRoutes.js (hunk 1 above)
Comment (alexisjanvier) [inline api/src/users/authenticateRoutes.js]: Or/and why not use `this` ? `this.userRepository`? Maybe have a `this.repositories = []` from the server.js, then `this.repositories.user` ?

**Comment by alexisjanvier** [comment]
:+1:

**Comment by alexisjanvier** [comment]
@djhi Must be rebased :(

**Pull request merged**